File: java/test/HelloWorld/HelloWorld.java
public class HelloWorld
{
	public static void main(String[] args)
	{
		System.out.println("Hello, World.");

		for(int i=0; i<args.length; ++i)
			System.out.println("Arg["+i+"]=\""+args[i]+"\"");

		Object
			o=null;

		if(o==null)
		{
			System.out.println("o==null");
		}
		else
		{
			System.out.println("o!=null");
		}

		if(o!=null)
		{
			System.out.println("o!=null");
		}
		else
		{
			System.out.println("o==null");
		}

		System.out.println(o==null?"o==null":"o!=null");
		System.out.println(o!=null?"o!=null":"o==null");
	}
}

File: c#/test/mono/HelloWorld/HelloWorld.cs
using System;
 
public class HelloWorld
{
	static public void Main (string[] args)
	{
		Console.WriteLine ("Hello Mono World");

		for (var i = 0; i < args.Length; ++i)
			Console.WriteLine("Arg[{0}]=\"{1}\"", i, args[i]);
	}
}
